README.md

```diff
@@ -72,19 +72,35 @@ Please visit the Amundsen installation documentation for a [quick start](https:/
 
 Please visit [Architecture](https://github.com/lyft/amundsen/blob/master/docs/architecture.md#architecture) for Amundsen architecture overview.
 
-## Supported connectors
-Amundsen supports two kinds of "nodes" in its graph today:
+## Supported Entities
 - Tables (from Databases)
 - People (from HR systems)
 - Dashboards
 
-Amundsen can connect to any database that provides `dbapi` or `sql_alchemy` interface (which most DBs provide).
-The databases we see most frequently used in the community are:
-- Hive and anything that works with Hive metastore (Spark SQL, Presto, Athena, etc.)
-- BigQuery
-- PostgreSQL and anything that uses the same interface (Redshift)
-- Snowflake
+## Supported Connectors
+
```

Comment:
IMHO this is a great place to mention what a "connector" is. Just in one line if possible, something like:
``` Connectors are .... . We support many connectors for either tables or dashboards, following there is a list of them.```

Reply:
yeah, I found the issue that somehow the markdown is broken in mkdocs(https://lyft.github.io/amundsen/ if it is not followed by bullet point but a list sentences... I haven't figured out why that's the case.

Reply:
@feng-tao it’s probably due to that the GitHub Markdown renderer is more forgiving than the Python based one in mkdocs. 

Perhaps some Markdown listing is needed like https://dev.to/jonasbn/blog-post-getting-to-know-github-actions-3m61 which has pointers/info to both a travis based setup and GitHub actions.

Reply:
thanks @jornh for the info!